Source/AwsCommonRuntimeKit/crt/Utilities.swift

```diff
@@ -81,15 +81,25 @@ extension Data {
 
 func withOptionalAWSByteCursorFromData<Result>(
     to data: Data?, _ body: (aws_byte_cursor) throws -> Result) rethrows -> Result {
-    guard let _data = data else {
+    guard let data = data else {
```

Comment:
nit: same for others if it's the same name
```suggestion
    guard let data else {
```